README.md

```diff
@@ -437,22 +439,22 @@ This is required, as the Authorization Code Grant needs to be redirected back to
 HTTP API token generation.
 
 #### Add your specified `irods_user_claim` to the user's claims
+
 Currently, the server looks for the custom claim in the ID Token, which is specified in the `irods_user_claim` parameter.
 This serves as the mapping mechanism for an OIDC User to an iRODS User.
 
 A user who authenticates but does not have the claim specified in `irods_user_claim` mapped in their account
-will not have access to the API.
-A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
+will not have access to the API.  A HTTP 400 Bad Request status code will be returned if the claim specified in `irods_user_claim` is not found.
```

Comment:
Collapse two spaces to one after period.

Reply:
dubious.   but okay.